src/main/java/Config/Config.java

```diff
@@ -0,0 +1,69 @@
+package Config;
+
+import Config.Routes.Route;
+import Config.Routes.RouteFactory;
+import Config.Routes.SimpleRouteFactory;
+import Controller.Handler.GetEchoHandle;
+import Controller.Handler.GetHandle;
+import Controller.Handler.Handler;
+import Controller.Handler.PostEchoHandle;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketException;
+import java.util.ArrayList;
+
+import static Config.Method.GET;
+import static Config.Method.POST;
+
+import static Validator.Validator.validArgsLength;
+import static Validator.Validator.validFlag;
+
+public class Config {
+  public static ArrayList<Route> routes = new ArrayList<>();
+  public static ArrayList<Handler> handlers = new ArrayList<>();
+
+  public static void createRoutes() {
+    SimpleRouteFactory factory = new SimpleRouteFactory();
+    RouteFactory routeFactory = new RouteFactory(factory);
```

Comment:
It was a good thought to use the decorator pattern here--how do you envision it working? Because right not it seems like RouteFactory just calls SimpleRouteFactory.createRoute. What will be the separate responsibilities of the `SimpleRouteFactory` and the `RouteFactory`?

Reply:
To add another question, what would this look like if you wrapped these two lines behind a static method call?  I.E., `RouteFactory.createSimple()`